packages/core/src/browser/shell/tab-bar-toolbar.tsx

```diff
@@ -106,7 +107,7 @@ export class TabBarToolbar extends ReactWidget {
     protected render(): React.ReactNode {
         return <React.Fragment>
             {this.renderMore()}
-            {[...this.inline.values()].map(item => TabBarToolbarItem.is(item) ? this.renderItem(item) : item.render(this.current))}
+            {[... this.inline.values()].map(item => TabBarToolbarItem.is(item) ? this.renderItem(item) : item.render(this.current))}
```

Comment:
Please revert unnecessary changes, it looks like your styling may be different than the projects and creating unnecessary changes.